TESTOrderScript.js

```diff
@@ -85,39 +102,46 @@ function addRow() {
         removeRowButton.classList.remove('noVisibility');
         removeRowLabel.classList.remove('noVisibility');
     }
+    // TODO: nit - unnecessary semi-colon
 };
 
 //Find the row number that the clicked element is in
 function findRowNumber(currentId) {
     const idOnDiv = $(currentId).closest('.order-fields__row').attr('id');
     const currentRowNumber = (idOnDiv.substring(3));
     return currentRowNumber;
+    // TODO: nit - unnecessary semi-colon
 };
 
 //If a remove row button is clicked, remove all fields in that row and set the previous row to the 'newest' row. If only one row remains, hide the remove option.
 function removeRow(currentId) {
     const currentRowNumber = findRowNumber(currentId);
+    // TODO: why parseFloat vs parseInt? Generally we try to avoid floats (and really anything in .js involving anything beyond basic math) because .js sucks at stuff like that
     const rowNumberMinusOne = parseFloat(currentRowNumber) - 1;
     const rowToAddClassTo = 'order-fields__row' + rowNumberMinusOne;
     const rowToRemove = 'order-fields__row' + currentRowNumber;
     //determine if the row that's going to be deleted has the 'newestRow' class on it
+    // TODO: as above, can we just use jQuery and a .css selector?
```

Comment:
Same issue as above. Does not work. Someone smarter than me needs to tell me why.